README-docker.md

```diff
@@ -26,7 +26,7 @@ images for you in **production mode**.
 ```
 cd deployment
 make build
-make run
+make runmake 
```

Comment:
Is this an accidental commit? I don't see any runmake target in the Makefile....

Reply:
Yes it was, ill be more steady from now onwards.